tests/MockDb.h

```diff
@@ -244,6 +245,22 @@ struct MockDbT : public sqlpp::connection
   {
     return {name};
   }
+
+  void start_transaction(sqlpp::isolation_level level)
+  {
+    _current_isolation_level = level;
```

Comment:
Storing the isolation level in the connection (even the mock) gives the wrong impression, I think.